app/celery/tasks.py

```diff
@@ -298,12 +303,11 @@ def save_smss(self, service_id: Optional[str], signed_notifications: List[Signed
 
     # we should move this check inside the for loop below
     check_service_over_daily_message_limit(KEY_TYPE_NORMAL, service)
-
     current_app.logger.info(f"Sending following sms notifications to AWS: {notification_id_queue.keys()}")
     for notification_obj in saved_notifications:
         try:
-            check_if_request_would_put_service_over_daily_sms_limit(KEY_TYPE_NORMAL, service, 0)  # type: ignore
-        except LiveServiceTooManySMSRequestsError:
+            check_service_over_daily_sms_limit(KEY_TYPE_NORMAL, service)
+        except (LiveServiceTooManySMSRequestsError, TrialServiceTooManyRequestsError):
```

Comment:
This was another potential bug where trial services were not covered by this?

Reply:
Yes - without the extra error getting caught here, a trial service could have sent above their limit in some cases.